config/config.yaml

```diff
@@ -164,8 +164,6 @@ reconstruction_settings:
     # TODO move to global
     reconstruction_dir: "output"
 
-  run: true
```

Comment:
I noticed in a separate thread on #72 that we might want to try incorporating a 'formal' deprecation system instead. Should we try this here instead of outright removing it?

Reply:
We still have a small enough user base that I am fine removing syntax or features. Deprecating them is a better practice but adds overhead for us.